v0.5/fastn-p2p/Cargo.toml

```diff
@@ -21,6 +21,9 @@ tokio.workspace = true
 tokio-util.workspace = true
 tracing.workspace = true
 
+# Context integration
+fastn-context = { path = "../../fastn-context" }
```

Comment:
This should say fastn-context.workspace = true, and path thing should be in workspace Cargo.toml